### DISCUSSION TITLE
New Addons for 1.18.2

### DISCUSSION THREAD

**Discussion by supermassimo** [DynamicTreesTeam/DynamicTrees#761] [Compatibility Addon Suggestions]
With Dynamic Trees for 1.18 being released, the door to 1.18 mod compatibility opens.
Following the same rules as the [1.12 suggestion list](https://github.com/DynamicTreesTeam/DynamicTrees/discussions/763), and the [1.16 suggestion list](https://github.com/DynamicTreesTeam/DynamicTrees/discussions/762), comment under this post 1.18 mods that could have Dynamic Trees compatibility.
Suggestions from the [discord server](https://discord.gg/83qcpkY) will also be added here.

**Please please please include how many trees a mod adds when suggesting**

> DO NOT suggest the following types of mods as they will not be considered:
>  - Mods without a public source (such as MCreator mods)

Those in **bold** were considered and/or are being worked on.
When a mod is finished and published, it is removed from this list.

**Update from previous version:** (add-ons from 1.12 or 1.16 that have yet to be updated)
- **[Pam’s HarvestCraft 2](https://www.curseforge.com/minecraft/mc-mods/pams-harvestcraft-2-trees)**
- **[Oh The Biomes You'll Go](https://www.curseforge.com/minecraft/mc-mods/oh-the-biomes-youll-go)**
- [Forbidden And Arcanus](https://www.curseforge.com/minecraft/mc-mods/forbidden-arcanus)
- [Mystical Biomes](https://www.curseforge.com/minecraft/mc-mods/mystical-biomes/)
- [Upgrade Aquatic](https://www.curseforge.com/minecraft/mc-mods/upgrade-aquatic)
- [Integrated Dynamics](https://www.curseforge.com/minecraft/mc-mods/integrated-dynamics)
- [Tech Reborn](https://www.curseforge.com/minecraft/mc-mods/techreborn)
- **[TerraFirmaCraft](https://www.curseforge.com/minecraft/mc-mods/terrafirmacraft)**
- [Tinker's Construct](https://www.curseforge.com/minecraft/mc-mods/tinkers-construct)
- [The Twilight Forest](https://www.curseforge.com/minecraft/mc-mods/the-twilight-forest)
- [Mystical World](https://legacy.curseforge.com/minecraft/mc-mods/mystical-world)

**New addons:**
- [Druidcraft](https://www.curseforge.com/minecraft/mc-mods/druidcraft)
- [William Wythers' Overhauled Overworld](https://www.curseforge.com/minecraft/mc-mods/william-wythers-overhauled-overworld) (Large scale)
- [Hexerei](https://www.curseforge.com/minecraft/mc-mods/hexerei)
- [Regions Unexplored](https://www.curseforge.com/minecraft/mc-mods/regions-unexplored) (Large scale)
- [Caupona](https://www.curseforge.com/minecraft/mc-mods/caupona)
- [Project Vibrant Journeys](https://www.curseforge.com/minecraft/mc-mods/project-vibrant-journeys) (Large scale)
- [TFC Florae](https://www.curseforge.com/minecraft/mc-mods/tfc-florae) (Large scale)
- [Architect's Palette](https://legacy.curseforge.com/minecraft/mc-mods/architects-palette)
- [Atmospheric](https://www.curseforge.com/minecraft/mc-mods/atmospheric)
- [Environmental](https://www.curseforge.com/minecraft/mc-mods/environmental)

**Comment by UHQ-GAMES** [comment]
Hey there! Just wanted to shoot my shot since it doesn't hurt to lol, but if I may suggest my mod Regions Unexplored. I don't know what other prerequisites are needed(like download count or things like that) but I have a public repo for the 1.18.2 version as well as a list of all the unique trees added by the mod (21 saplings, approx. 70 distinguishable trees in worldgen). I will also provide a link to the 1.18 repo below. This is something id been wanting to do myself but can't due to not having enough 
time so thought id throw out the suggestion, thanks for reading!

https://github.com/UHQ-GAMES/Regions_Unexplored_Forge_1.18.2

**Comment by Jrassic** [comment]
I would love an addon for Croptopia (https://www.curseforge.com/minecraft/mc-mods/croptopia) is has 26 (Fruit)Trees. 
Thank you for your consideration!

**Comment by CupcakeJemon601** [comment]
I’d like to suggest another mod as well for DT is Project Vibrant Journeys. I think this would be a splendid addition to DT 
https://www.curseforge.com/minecraft/mc-mods/project-vibrant-journeys

**Comment by DJBaphomet** [comment]
Would love to see compat for [Architect's Palette](https://legacy.curseforge.com/minecraft/mc-mods/architects-palette). It only has 1 tree, which just reuses the vanilla acacia growth, and it doesn't generate naturally in-world (may change later on)

**Comment by GrandmaPork** [comment]
The Quark addon needs dynamic glow shrooms

**Reply by supermassimo** [reply]
That is already under development :)

**Reply by GrandmaPork** [reply]
🙌🏽🙌🏽

**Comment by MidasMischief** [comment]
An addon for Atmospheric would be epic! ^^
https://www.curseforge.com/minecraft/mc-mods/atmospheric

**Reply by ColonelGerdauf** [reply]
Add Environmental to that as well

**Reply by bigj1525** [reply]
I agree!

**Comment by RedAngel121** [comment]
Would be nice to see Incendium with compat in 1.18.2
https://www.curseforge.com/minecraft/mc-mods/incendium

**Comment by OceanSnipe17** [comment]
It would be cool to have Nyctophobia added to the list. A neat small mod that just adds a few biomes and some spooky biome designs. Most of its trees are spruce. Some are stripped spruce though and also a good chunk have spiderwebs in them.

https://www.curseforge.com/minecraft/mc-mods/nyctophobia-forge

**Comment by Zynara** [comment]
Is there, or could you add compatibility with mods such as "Falling Tree" or other mods that bring the trees down with a single chop?

**Comment by triggeredfloof** [comment]
idk if this is the right place for this or not but this is what came up on google. but i just tested dynamic trees in 1.21.1 with [William Wythers' Overhauled Overworld](https://www.curseforge.com/minecraft/mc-mods/william-wythers-overhauled-overworld) and its not working how i thought it would. i just get a mix of normal trees and dynamic trees. if there is a better place for me to go about this then can you point me in the right direction
<img width="1377" height="770" alt="image" src="https://github.com/user-attachments/assets/fb98d5df-23df-40ef-9f67-5fa3ff7df2b0" />
